### ISSUE TITLE
Fix std Err and  Output of Bule regarding pipe from Solvers

### ISSUE THREAD

**Issue by vale1410** [vale1410/bule#10]
Currently the std.output of the solvers is piped to stderr from bule. Fix.

**Comment by AbdallahS** [comment]
I don't think this issue occurs with the reimplementation. Can you check and close if it works the way you intended?
